mypy/checker.py

```diff
@@ -1179,6 +1190,14 @@ def visit_assignment_stmt(self, s: AssignmentStmt) -> None:
         """
         self.check_assignment(s.lvalues[-1], s.rvalue, s.type is None, s.new_syntax)
 
+        if (s.type is not None and
+                self.options.disallow_implicit_any_types and
+                has_any_from_silent_import(s.type)):
+            if isinstance(s.lvalues[-1], TupleExpr):  # is multiple
```

Comment:
Nitpick: this comment should be a bit more explicit.  In general, I'd err on the side of trying to be very very clear in comments, even if that feels a little verbose.  In this case, I'd consider moving the comment to the next line and writing something like:
```
# This is a multiple assignment. Instead of figuring out which type is problematic, give a generic error message.
```